### PR TITLE
Bug 1373726 - Base bundle identifier incorrectly generated for Nighty builds

### PR DESCRIPTION
**Pull request by st3fan**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
AppInfo.swift | 9 | Force casts should be avoided. 
AppInfo.swift | 13 | Force casts should be avoided. 
AppInfo.swift | 17 | Force casts should be avoided. 
AppInfo.swift | 54 | Force casts should be avoided. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -6310,10 +6310,11 @@
 			buildSettings = {
 				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Extensions/Entitlements/FennecEnterprise.entitlements";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 9G8J6YA743;
```
Comment (garvankeeley) [inline Client.xcodeproj/project.pbxproj]: @st3fan was this accidental commit?

Comment (st3fan) [reply Client.xcodeproj/project.pbxproj]: @garvankeeley looks like it - but this commit is months old .. is it causing trouble?

Comment (garvankeeley) [reply Client.xcodeproj/project.pbxproj]: i am in one of those hellish states where xcode suddenly won't run on my device "iPhone has denied the launch request", for like an hour now. I am just grasping at straws here.

Comment (garvankeeley) [reply Client.xcodeproj/project.pbxproj]: to clarify: this is not the problem, i just noticed it when looking for other problems

**Comment by garvankeeley** [comment]
i see this error when update signing happens
screen shot 2017-09-07 at 2 20 49 pm

**Comment by garvankeeley** [comment]
i see this error when update signing happens
screen shot 2017-09-07 at 2 20 49 pm

**Pull request merged**